clouddq/main.py

```diff
@@ -337,8 +283,18 @@ def main(  # noqa: C901
         dbt_path = dbt_runner.get_dbt_path()
         dbt_rule_binding_views_path = dbt_runner.get_rule_binding_view_path()
         dbt_entity_summary_path = dbt_runner.get_entity_summary_path()
-        dbt_profiles_dir = dbt_runner.get_dbt_profiles_dir()
-        environment_target = dbt_runner.get_dbt_environment_target()
+        dbt_profiles_dir = dbt_runner.get_dbt_profiles_dir(
```

Comment:
If I am not mistaken, `_resolve_connection_configs` is effectively called twice (once in `get_dbt_profiles_dir` and once in `get_dbt_environment_target`). Why do we need to do this? Or it means we should refactor the code?

Reply:
I think, we should refactor the code a bit.